src/components/IndexTable/IndexTable.tsx

```diff
@@ -580,6 +646,7 @@ function IndexTableBase({
         className={stickyHeadingClassName}
         key={heading.title}
         style={headingStyle}
+        data-index-table-sticky-heading="1"
```

Comment:
```suggestion
        data-index-table-sticky-heading
```

Nitpick: if we're not using the `1` we could default it to true